components/arm/fake/fake.go

```diff
@@ -105,7 +100,7 @@ func NewArm(cfg config.Component, logger golog.Logger) (arm.LocalArm, error) {
 	case modelPath != "":
 		model, err = referenceframe.ModelFromPath(modelPath, cfg.Name)
 	default:
-		err = errAttrCfgMissing
+		model = referenceframe.NewSimpleModel(cfg.Name)
```

Comment:
Add a comment saying "if no arm model is specified, an empty one with 0 dof and 0 spatial transformation is created" or something along those lines.